ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -34,20 +34,39 @@ const PADDED_INLINE_STYLE = {
   ...INLINE_STYLE,
 }
 
-const BaseGeneLabelContent = styled(({ color, customColor, label, maxWidth, containerStyle, dispatch, ...props }) => {
+const BaseGeneLabelContent = styled(({
+  color, customColor, detail, label, maxWidth, containerStyle, dispatch, ...props
+}) => {
   const labelProps = {
     ...props,
     size: 'mini',
-    content: label,
+    content: <span>{label}</span>,
+    detail: detail || null,
```

Comment:
If the prop is not being renamed then we should just let it pass through in the `... props`, no need to destructure it at all

Reply:
Detail needs to be transformed to null, or it will be added to every label and the brackets () will be on all of them. I guess it could be set to null earlier and this line could be removed.